string command_type
float64[] joint_positions
float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
float64 duration
bool relative
string gripper_state
